<launch>
  <!-- Octomap Traversability Node -->
  <node pkg="aggressive-navigation" type="octomap_traversability_node" name="octomap_traversability"
	output="screen"
	launch-prefix="gdb -ex run --args" >
	<!-- launch-prefix="valgrind -\-tool=memcheck -\-db-attach=yes" >  -->
    
    <param name="resolution" value="1" />
    
    <!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
    <param name="frame_id" type="string" value="map" />
    
    <!-- maximum range to integrate (speedup!) -->
    <param name="sensor_model/max_range" value="300.0" />
    
    <!-- data source to integrate (PointCloud2) -->
    <remap from="cloud_in" to="/velodyne_points" />

    <param name="track_changes" type="bool" value="true" />
    <param name="topic_changes" type="string" value="/octo_changes" />
    <param name="change_id_frame" type="string" value="/map" />

    <param name="robot_length" type="double" value="0.5" />
    <param name="robot_width" type="double" value="0.5" />
    <param name="robot_height" type="double" value="0.5" />
    
  </node>
</launch>
